<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SketcherWidgetForm</class>
 <widget class="QWidget" name="SketcherWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D Sketcher</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="schrodinger::sketcher::SketcherSideBar" name="side_bar_wdg" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="schrodinger::sketcher::SketcherTopBar" name="top_bar_wdg" native="true"/>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="schrodinger::sketcher::View" name="view">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="carbon_labels_combo"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="font_sb"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::View</class>
   <extends>QGraphicsView</extends>
   <header>schrodinger/sketcher/molviewer/view.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::SketcherTopBar</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/sketcher_top_bar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::SketcherSideBar</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/sketcher_side_bar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
</ui>
